callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
totally! already working on subclassing the PBKDF2 hasher right now, will add the license clause too :)